api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/CardClient.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.dhsc.htbhf.claimant.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.dhsc.htbhf.claimant.exception.CardClientException;
+import uk.gov.dhsc.htbhf.claimant.model.card.CardRequest;
+import uk.gov.dhsc.htbhf.claimant.model.card.CardResponse;
+
+import static uk.gov.dhsc.htbhf.claimant.SchedulerConfig.SCHEDULER_REST_TEMPLATE_QUALIFIER;
+
+/**
+ * Service for interacting with the card services api.
+ */
+@Service
+@Slf4j
+public class CardClient {
+
+    private static final String CARDS_ENDPOINT = "/v1/cards";
+    private final String cardsUri;
+    private final RestTemplate restTemplate;
+
+    /**
+     * Uses a rest template which sets the session id to the current job id.
+     *
+     * @param baseUri      base url of the card services api.
+     * @param restTemplate rest template which sets the session id to the current job id.
+     */
+    public CardClient(@Value("${card.base-uri}") String baseUri,
+                      @Qualifier(SCHEDULER_REST_TEMPLATE_QUALIFIER) RestTemplate restTemplate) {
+        this.cardsUri = baseUri + CARDS_ENDPOINT;
+        this.restTemplate = restTemplate;
+    }
+
+    public CardResponse createNewCardRequest(CardRequest cardRequest) {
```

Comment:
this should probably be called 'requestNewCard' or 'createNewCard'

Reply:
Renamed to requestNewCard